Content.Client/CriminalRecords/CriminalRecordsConsoleWindow.xaml

```diff
@@ -1,36 +1,152 @@
+                        <TextureRect Name="PersonJobIcon"
+                                     TextureScale="2 2"
+                                     Margin="6 0"
+                                     VerticalAlignment="Center" />
+                        <Label Name="PersonJob" />
+                    </BoxContainer>
+                    <BoxContainer Orientation="Horizontal"
+                                  Margin="0 0 0 5">
+                        <Label Text="{Loc 'general-station-record-prints-filter'}"
+                               FontColorOverride="DarkGray" />
+                        <Label Text=":"
+                               Margin="0 0 6 0"
+                               FontColorOverride="DarkGray" />
+                        <Label Name="PersonPrints" />
+                    </BoxContainer>
+                    <BoxContainer Orientation="Horizontal"
+                                  Margin="0 0 0 5">
+                        <Label Text="{Loc 'general-station-record-dna-filter'}"
+                               FontColorOverride="DarkGray" />
+                        <Label Text=":"
+                               Margin="0 0 6 0"
+                               FontColorOverride="DarkGray" />
+                        <Label Name="PersonDna" />
+                    </BoxContainer>
+                    <PanelContainer StyleClasses="LowDivider"
+                                    Margin="0 5 0 5" />
+                    <BoxContainer Orientation="Horizontal"
+                                  Margin="0 5 0 5">
+                        <Label Name="StatusLabel"
+                               Text="{Loc 'criminal-records-console-status'}"
+                               FontColorOverride="DarkGray" />
+                        <Label Text=":"
+                               FontColorOverride="DarkGray" />
+                        <Label Name="PersonStatus"
+                               FontColorOverride="DarkGray" />
+                        <AnimatedTextureRect Name="PersonStatusTX"
+                                             Margin="8 0" />
+                        <OptionButton Name="StatusOptionButton"
+                                      MinWidth="130" />
+                        <!-- Populated in constructor -->
+                    </BoxContainer>
+                    <RichTextLabel Name="WantedReason"
+                                   Visible="False"
+                                   MaxWidth="425" />
+                    <Button Name="HistoryButton"
+                            Text="{Loc 'criminal-records-console-crime-history'}"
+                            Margin="0 5" />
                 </BoxContainer>
-                <RichTextLabel Name="WantedReason" Visible="False"/>
-                <Button Name="HistoryButton" Text="{Loc 'criminal-records-console-crime-history'}"/>
+            </BoxContainer>
+            <BoxContainer Orientation="Horizontal"
```

Comment:
I'd like to see this container shrunk down so it matches the width of the crew list above it. Consider using a drop-down list for the buttons (when these labels are translated into Russian, for example, they are quite long)